backend/src/main/java/com/summerHack/diningTogether/repository/UserRepository.java

```diff
@@ -11,4 +12,7 @@ public interface UserRepository extends JpaRepository<User, Long> {
     Optional<User> findByEmail(String email);
 
     Optional<User> findByUsername(String username);
+
+    @Query("SELECT u FROM User u WHERE u.verificationCode = ?1")
+    public User findByVerificationCode(String code);
```

Comment:
I guess that you don't have to add the annotation if you use `public Optional<User> findByVerificationCode(String code)`